CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Fixed
+- `title` prop to `Image` component
```

Comment:
Isn't this an addition? 
```suggestion
### Added
- `title` prop to `Image` component
```

Reply:
Already exists a log with the Added mark (## [0.4.0] - 2020-02-10). The props was there but it wasn't being passed correctly